### PR TITLE
New Kinetis targets

### PR DESCRIPTION
**Pull request by flit**
A whole bunch of new Kinetis targets. All board IDs are officially registered.

- FRDM-KE15Z (0213)
- FRDM-KL27Z (0261)
- FRDM-KL28Z (0224)
- FRDM-KL43Z (0262)
- FRDM-KL82Z (0218)
- FRDM-K28F (0224)
- FRDM-K66F (0311)
- FRDM-K82F (0217)
- TWR-KL28Z72M (0203)
- TWR-KE18F (0206)
- HVP-KE18F (0216)

- Corrected board IDs for FRDM-KL02Z (0204) and FRDM-KW24F (0250).

### CODE REVIEW

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Comment by c1728p9** [comment]
@flit, the tests are failing because the target 'FRDM-KL82Z' and likely others can't be built from mbed-os yet. You'll need to replace targets like this with 'None' to get testing to pass. I would just leave a comment as to what the target should be once supported.

**Comment by flit** [comment]
@c1728p9 Ok, thanks. Will fix shortly.

**Comment by flit** [comment]
@c1728p9 I removed the target name for FRDM-K82F, FRDM-KL82Z, and FRDM-K28F. Do you know if FRDM-KL27Z or FRDM-KL43Z were problematic? Where can I check if a target officially has mbed-os support?

**Comment by c1728p9** [comment]
I don't know offhand. If you run testing locally on any target it use the compile API to build all targets.

**Comment by flit** [comment]
@c1728p9  I'll go ahead and remove them since they aren't listed on the mbed.org boards list.

**Comment by c1728p9** [comment]
If you can select them as a target in the online IDE then you can leave them

**Comment by flit** [comment]
Adding a platform in the online compiler takes you to the same boards list. So they're out.

File: test/info.py [hunk 1]
```diff
@@ -78,17 +78,17 @@
     (   0x210,      'k20dx_frdmkl05z_if',                       'k20dx_bl',         'FRDM-KL05Z'                            ),
     (   0x213,      'k20dx_frdmke15z_if',                       'k20dx_bl',         None                                    ),
     (   0x216,      'k20dx_hvpke18f_if',                        'k20dx_bl',         None                                    ),
-    (   0x217,      'k20dx_frdmk82f_if',                        'k20dx_bl',         'FRDM-K82F'                             ),
+    (   0x217,      'k20dx_frdmk82f_if',                        'k20dx_bl',         None                                    ),
```
Comment (c1728p9) [inline test/info.py]: Might not hurt to have a comment indicating what they will be when supported

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Comment by c1728p9** [comment]
Thanks for the update

**Comment by flit** [comment]
@maclobdell Thanks. Two questions: the example output you show lists, for example, "K22F" whereas in the DAPLink info.py that board has the mbed-os target name of "FRDM-K22F". Also, it shows that KL82Z is supported but that board caused a test failure.

**Comment by maclobdell** [comment]
@c1728p9  Can you explain that?

**Comment by c1728p9** [comment]
The name should match that in the url on the mbed platform page for example https://developer.mbed.org/platforms/FRDM-KL46Z/ would be **FRMD-KL46Z**

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Pull request merged**